api/app/kobo.py

```diff
@@ -162,32 +164,18 @@ def parse_datetime_params(
 
 
 def get_responses_from_kobo(
-    form_url: str, auth: tuple[str, str], form_name: str
+    form_url: str, auth: tuple[str, str], form_id: str
 ) -> tuple[Any, dict[str, str]]:
     """
     Request kobo api to collect all the information related to a form.
 
     Also, retrieve the form responses for parsing and filtering.
     """
-    # do not send empty credentials, skip auth altogether if it's not available
-    if auth == ("", ""):
```

Comment:
This check for empty strings has been removed in this patch. There is only a check for `None` values in `get_kobo_params`. Is it worth checking for empty strings as well, to avoid a possible error from the kobo server?

Reply:
I wrote a better way to handle empty strings here: 85170c9fc6e208c754ed314c9acb087a96171b32